creyPY/fastapi/pagination.py

```diff
@@ -94,8 +95,8 @@ def unwrap_scalars(
     return [item[0] if force_unwrap else item for item in items]
 
 
-def paginate(
-    connection: Session,
+async def paginate(
+    connection: Union[Session, AsyncSession],
```

Comment:
Please make this backwards compatible. I don't think that this paginate function can be called synchronously right?